tidb-in-kubernetes: add one faq item (#1609)
pingcap/docs: dev/tidb-in-kubernetes/faq.md

    ```shell
    kubectl exec -it <tidb-pod-name> -c tidb -n <namespace> -- cat /etc/tidb/tidb.toml
    ```

## Why does TiDB Operator fail to schedule Pods when I deploy the TiDB clusters?

Three possible reasons:

> * Insufficient resources or HA Policy causes the Pod stuck in the `Pending` state. Refer to [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state) for more details.

Reviewer: ```suggestion
* Insufficient resource or HA Policy causes the Pod stuck in the `Pending` state. Refer to [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state) for more details.
```

> * Insufficient resources or HA Policy, which causes the Pod in the `Pending` state. Refer to the details in [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state).

Reviewer: ```suggestion
* Insufficient resources or HA Policy causes the Pod stuck in the `Pending` state. Refer to [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state) for more details.
```

> * The Pod is in the `Pending` state, which caused by insufficient resources or HA Policy. Refer to the details in [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state).

Reviewer 2: ```suggestion
* Insufficient resources or HA Policy, causing the Pod in the `Pending` state. Refer to the details in [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state).
```

> * The `taint` property are applied to some Nodes, which refuses the Pod to be scheduled unless that Pod has a matching `toleration`. Refer to the details in [taint & toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/).

Reviewer 2: ```suggestion
* The `taint` property is applied to some Nodes, which refuses the Pod to be scheduled unless that Pod has a matching `toleration`. Refer to the details in [taint & toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/).
```

> * The Pod is stuck in the `ContainerCreating` state and causes scheduling failure. In such case, you can check if there is more than one TiDB Operator deployed in the Kubernetes cluster. Conflicts occur when custom schedulers in multiple TiDB Operators schedule the same Pod in different phases.

Reviewer 2: ```suggestion
* Scheduling conflict, which causes the Pod stuck in the `ContainerCreating` state. In such case, you can check if there is more than one TiDB Operator deployed in the Kubernetes cluster. Conflicts occur when custom schedulers in multiple TiDB Operators schedule the same Pod in different phases.
```

>     You can execute the following command to verify whether there are more than one TiDB Operator deployed. If more than one record is returned, delete extra TiDB Operator to fix scheduling failure.

Reviewer 2: ```suggestion
    You can execute the following command to verify whether there are more than one TiDB Operator deployed. If more than one record is returned, delete extra TiDB Operator to fix scheduling conflict.
```

>     You can execute the following command to verify whether there are more than one TiDB Operator deployed. If more than one record is returned, delete extra TiDB Operator to fix scheduling conflict.

Reviewer: ```suggestion
    You can execute the following command to verify whether there is more than one TiDB Operator deployed. If more than one record is returned, delete the extra TiDB Operator to resolve the scheduling conflict.
```
